src/main/CommandHandler.cpp

```diff
@@ -70,14 +70,10 @@ CommandHandler::CommandHandler(Application& app) : mApp(app)
 
     mServer->add404(std::bind(&CommandHandler::fileNotFound, this, _1, _2));
 
-    if (mApp.modeHasDatabase())
-    {
-        addRoute("dropcursor", &CommandHandler::dropcursor);
-        addRoute("getcursor", &CommandHandler::getcursor);
-        addRoute("setcursor", &CommandHandler::setcursor);
-        addRoute("maintenance", &CommandHandler::maintenance);
-    }
-
+    addRoute("dropcursor", &CommandHandler::dropcursor);
```

Comment:
why re-enable this? it should be disabled if we don't have historical data enabled